src/compose/ComposeText.js

```diff
@@ -56,17 +56,16 @@ class ComposeText extends React.Component {
       contentHeight: MIN_HEIGHT,
     };
   }
-
   handleSend = () => {
     const { auth, narrow } = this.props;
     const { text } = this.state;
-
+    const message = replaceEmojiWithTextCode(text);
```

Comment:
Not sure if we want to replace this here. Maybe the backend will prefer it gets the text raw and convert it there. (it is not yet implemented though) @timabbott what do you think?